training/index.html

```diff
@@ -0,0 +1,585 @@
+      Conversions</h2>
+    <ul class="dotless ul-unindent">
+      <li class="text-lg text-left">1 Sq. Mile</li>
+      <li>= Nominal Section</li>
+      <li>= 640 acres</li>
+      <li>= 2.590 sq. km.</li>
+      <li>= 259 hectares</li>
+    </ul>
+    <ul class="dotless ul-unindent">
+      <li class="text-lg text-left">1 Acre</li>
+      <li>= 10 sq. chains</li>
+      <li>= 160 sq. rods</li>
+      <li>= 160 perches</li>
+      <li>= 160 poles</li>
+      <li>= 43,560 sq. feet</li>
+      <li>= 0.4047 hectare</li>
+      <li>= 4,047 sq. meters</li>
+    </ul>
+    <ul class="dotless ul-unindent">
+      <li class="text-lg text-left">1 Hectare</li>
+      <li>= 2.47 acres</li>
+    </ul>
+    <ul class="dotless ul-unindent">
+      <li class="text-lg text-left">1 Sq. Meter</li>
+      <li>= 10.764 sq. ft.</li>
+    </ul>
+  </section>
+  <section class="point-ids text-md">
+    <p>XxxYyy, where Xxx = the following range of values</p>
+    <ul class="dotless ul-unindent">
+      <li>711 - 719 Town Sites, City Boundaries</li>
+      <li>720 - 739 County Boundaries</li>
+      <li>740 - 769 State Boundaries</li>
+      <li>770 - 799 Forest Service, Parks Service, Reservations,</li>
+      <li>Grant, Other Boundaries with Mileposts.</li>
+      <li>800 - 836 Meanders (right bank even #, left bank odd #.)</li>
+      <li>837 - 879 Tracts</li>
+      <li>880 - 899 Right of Ways</li>
+      <li>900 - 999 Mineral Surveys, Homestead Entries, etc.</li>
+    </ul>
+    <p>Yyy - Numbered sequentially along the survey in increments of 10. The prefix</p>
+    <p>Xxx - may be the same for all points in a similar survey.</p>
+  </section>
+  <section class="id-irregularities text-md">
+    <p>Irregularities of Townships for ones places of Xxx and Yyy.</p>
+    <p>1, 2, 8 or 9 -> offset corners along interior section lines and section subdivision lines where +/- 2 is the
+      recommended naming scheme and THE GRAPGICS RULE MUST BE FOLLOWED.</p>
+    <p>3 or 7 -> offset corners along township boundaries, angle points, witness corners, and witness points.</p>
+    <p>4 or 6 -> intersections of rectangular/special surveys and 1/16th corners of elongated sections.</p>
+    <p>0 -> all regular aliquot corners down to 1/16th.</p>
+    <p>5 -> all regular aliquot corners down to 1/256th.</p>
+  </section>
+  <img class="township-img" src="images/township.png" />
+  <img class="section-img" src="images/quarter-section.png" />
+  <img class="corner-img" src="images/quarter-quarter-section.png" />
+  <section class="blm-label bg-gray text-right grid--center" style="padding-right: .5em;">
+    <div>Bureau of Land</div>
+    <div>Management (BLM)</div>
+    <div>Point IDs</div>
+  </section>
+  <section class="blm-id bg-gray grid--center" style="margin-left: -.5em;padding:0 .5em;">
+    <img src="images/blm-point-id.png" style="width: 100%;border: 1px solid gray;" loading="lazy" />
```

Comment:
Looks like the underlined title of this image is cropped-out on the pdf version, possibly because it's redundant with the gray box text. **Suggestion:** crop out `Bureau of Land Management (BLM) Point ID's` then make image larger to fill the gray box space that will be result from the cropping.